collect.sh

```diff
@@ -17,104 +19,38 @@ log_verbose() {
   fi
 }
 
-rm -rf data/$PROCESS
```

Comment:
Need to clean up the testbed before start new one. To make sure no noise is introduced.